<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelViewOptionsPage</class>
 <widget class="QWidget" name="ModelViewOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>3D model:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Background image:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Utils::PathChooser" name="backgroundPathChooser" native="true">
     <property name="toolTip">
      <string>Select the image that is shown in the background.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Utils::PathChooser" name="modelPathChooser" native="true">
     <property name="toolTip">
      <string>Select the 3D model file here.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="enableVbo">
     <property name="toolTip">
      <string>Click to enable the use of Vertical Blanking.
It is not enabled by default because some graphic cards do not
support it, and crash the GCS. Enabling improves performance, though, so you can experiment at your own risk.</string>
     </property>
     <property name="whatsThis">
      <string>VBO allow for performance gains for GPUs that support it (most cards).  This may cause cards with faulty drivers to crash.</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Enable VBOs:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header>utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
